xil_cloud_telematics/.env

```diff
@@ -1,3 +1,3 @@
-DOCKER_ORG=usdotfhwastolcandidate
-DOCKER_TAG=neon
-OPS_DOCKER_TAG=release-neon
+DOCKER_ORG=usdotfhwastol
+DOCKER_TAG=carma-system-4.7.0
```

Comment:
Why is this docker tag 4.7.0? Is this tag only for carma-config image? Or all images in the docker-compose.yml file?

Reply:
we are only targeting the cda-telematics image with new version 4.7.0, Since there are some unreleased platform changes in both Miura and Neon release branches we are pointing other docker files to 4.5.0 and not publishing them on docker hub(disabled other docker automated builds except cda-telematics).

Reply:
An additional note on this, this environment variable is not used for the carma-config image, it is used for the services in the docker-compose.yml. In this xil_cloud_telematics config the DOCKER_ORG tag is only read by the ros2-nats-bridge and streets-nats-bridge images. The other services use the latest release which is hardcoded to 4.5.0